genesis/checkpoints.json

```diff
@@ -0,0 +1,612 @@
+            "9JT71povEckWQvc8iwgoZmxNKhonmeBiQcGeG5uD8aSxzigJE",
+            "9TFR2TmyvJvsa8qUdED2K1iGvDu6iMUWE7rmt6cKamQiVp8nW",
+            "CULPvaK6i2C7V4eRnQqKosMjzDc2YeAbRrXqzc5nEYRRg5byF",
+            "CoiAuuPMNhVBGytjsM2D6QTSqAFkb9mbAaeJ8EELHrP78wyDo",
+            "EGHXu22WF9mSHfkmQGrcFa78aGKNpQHrf3kQMXvkroTp4KG57",
+            "FyuoVPbTvmAwoMMvauGsLu25QkRT2ffCtuveAAZGFnfrFXpGR",
+            "GhTxU8m3fkBzN1a1fxfnV4oFPACqAZfnu3iayiEMcV733yq3N",
+            "HKEFB2cTrLc3jq8Eevp5gNK2W783bgJntPT9LetzRThK4GWbF",
+            "LU9mnCVtCKVAh786Unb8rR53WWSWw5TaEfs5TkEhxZm1kDumW",
+            "MDAux1a22nQMTU2yNVgyrbDoPU5Ym37kFh3nSqtBKNSbu7B5U",
+            "MJbvKe3mG4PuCrGrgiKCmUiJqMW5iFbaYzLkPG3o6oF9sDb3S",
+            "MWeuiyZC2dovgSWxHwD2dtJwZ4bBb6kynrUH7qcwK7T9JDhSe",
+            "NjwaUveSc6nfRZdwsLp7LgDJCNXKTbwV5XusjK2wyxXNqtn4j",
+            "NmCqVhQWBbh9ogkWjoeVqWGgkHKTtosxoDh2nPdEWZ8AQfwgj",
+            "TcNkLFGvbk1oSy3BsR655T4irnvXq2o5aFcRSjV9vx3haPpoM",
+            "TgfoQbxhbvH5ukjUQz2aoZVau4p9zxYi1afYjHPCWuDmSbTG9",
+            "TqSP2Qwdcf6mmoLAwmh2B4PSZ8T5L6kbWuaVNFqHfUcTN9vLE",
+            "TxAbe5TanwQyh3yLiMjLfjZRE7GWhGqsybixCDFhppJZ79qzr",
+            "UWQ9xMgsmmnAnsLPPqnEAR8Qco5hEVD8H8WGMYtAAVzGvRCHy",
+            "YEjyUmUMva7AX8T2eUv2JrHeLf6cxjSppZN34XgWqGVkKSKER",
+            "YXZEoXrp68wmFg6S9i84YHkRcLJPrZEsT4oQGWipgwZArD54A",
+            "ZKskrRDWRWzYBk5zEEanuKJbYjxaVSg1FzJxKVuDT7LyYtmSe",
+            "ZwQH5ki9kwVVE5Qpeg6tSLF6ciid5Qtbg7HuJnYsr9RvpQTFV",
+            "a3cziexWAoPKNpue1yvsQNrLPCuNmbgxmaX7wP3cbYGfkTxYv",
+            "bC1uQUxr4fTk6WXqwhgMecBtGFWw3Uyzuk68GHZkn19k1S8rd",
+            "bQdKidaawr1NZTnbrEnAJn46TB7t8qyfnAxgDH47M93XuyQY7",
+            "bZxvvKp5GuwzDed4LgngN3rkdxxjnRV9ZMWJZmLteq2CiYfsj",
+            "baqCaeXpD886QdQPtfqBUHVq34KuiDPEcw1TDyDHHFS3PPUYe",
+            "c2TftZBnzAk2iE7QhNMeYGxG4cYK7p3S4WVHx8yWFDx38FtSz",
+            "c36RHow7SjbGHwkoy2yEeBnq8ZUiPqPndHxAYgEWu5pHrFV8G",
+            "cZK53Cyf7WsNch1arhdJbMUwGHu4NPAt32fQDCmePWcsDNrnN",
+            "egZE8zk5DFhGjfbRyCHV2hLAXh2NqXzpQN4HBKA8wYMuNjjFB",
+            "fw4ZYUiKbkenFZgGeszwKLiu7VkQKvhy2FPpmsH1ZYiAA43eS",
+            "g2wo44qqYqakCM4NDgYSo1YEQknWq9K92QP62SaNrExqGg3dX",
+            "hmUpps6g13vAiorcK3fJtaXeCs9C4pzygk269XoNpYWE9pwLA",
+            "hnV2YHfTxCBcjqwSzYDNBKTikiqiDfDsn3s4ypyukXog2bLyh",
+            "iGQfwhtJ8aKpatZtgHTgJDUAEzMuwePMbRGk86fSmTieNLn6",
+            "khWVWX8eL4ahGJBd4E7GySRXenfbF4bqX79ZPuMz5K7rB5P7j",
+            "nHR81kNRdwvJ66ZxyaT2dZ6pgJtET32aEWixk62W6QXT5ZR28",
+            "otVKKrh7Dkvpmd8BbnXiCh1ivnnvJLXft48ajHxduSEbHrshT",
+            "qvuGSY6oPt8a2jYPvtHPVKmR4Hg5K77rD2J84twVMxYhfZLHp",
+            "sp2ec7ivV4BheoGoBm7RUpjGnvz4AKruvtq2G9ekmJWRUKNuw",
+            "tNM6B5Wt2JJ1DXhChZbucfckBwwTxwX9fv7ZwBppsCifpqneh",
+            "v78MgFnTuxBm8hwYhLkjphFXqdSZaAHPJXwgpcHGbLYcKnrF8",
+            "zGhQDuTyFxZrY3NE3QzBU4v6r8RphEJfMPUgZ2cNHuC7xHbcB"
+        ]
+    }
+}
```

Comment:
nit: missing newline at EOF (or was this done intentionally if it doesn't play nice w/ goembed?)

Reply:
This file was auto-generated. If we want to add a newline here, we should do it in the script.